modules/ocf_mirrors/files/healthcheck

```diff
@@ -53,6 +57,19 @@ def get_updated_unix_timestamp(mirror_url):
     return datetime.utcfromtimestamp(int(req.text.split()[0]))
 
 
+def get_updated_recursive_ls(mirror_url):
+    """Find the time the host was last synced.
+    Should contain the output of ls -lR
```

Comment:
Maybe mention that this selects the most recent date from the output of `ls -lR`?